### DISCUSSION TITLE
[python] errno-13-permission-denied  (ps: could not read the file)

### DISCUSSION THREAD

**Discussion by SilverEdge99** [pyinstaller/pyinstaller#8574] [Help]
Hi guys
I am a beginner at learning python. I encounter a problem when I try to use 
**f = open(r"C:\Users\XXXX\Desktop\record.text")**   to open a file on Desktop. 
And the errors come out:  Traceback (most recent call last):
  File "<pyshell#168>", line 1, in <module>
    f = open(r"C:Users\XXXX\OneDrive\Desktop")
PermissionError: [Errno 13] Permission denied: 'C:Users\\XXXX\\OneDrive\\Desktop'

I tried to check the permission on text security:
![image](https://github.com/bpyinstaller/pyinstaller/assets/117233077/bc55a91a-3ccb-4f23-9fe9-72bac3896d57)

Can someone help, and give a solution to this problem?

**Comment by SilverEdge99** [comment]
![image](https://github.com/pyinstaller/pyinstaller/assets/117233077/144d8d27-d70e-43dc-9acf-9f5ede64e852)

**Comment by bwoodsend** [comment]
We're not a _fix my code_ service. This question isn't even about PyInstaller.
